### PR TITLE
Emojis for activities, food, drink, people (~100 more)

### PR DESCRIPTION
**Pull request by wachunga**

### CODE REVIEW

File: public/emojis.js [hunk 1]
```diff
@@ -434,43 +533,56 @@ const emojis = {
...
   TRUCK: "🚙",
-  TRUMPET: "🎺",
   TUBE: "🧪",
   TULIP: "🌷",
+  TURBAN: "👳",
   TURTLE: "🐢",
   TYPE: "⌨️",
   UFOS: "🛸",
-  UNICORN: "🦄",
+  UNICORN: "🦄", // but how?
```
Comment (wachunga) [inline public/emojis.js:548]: why include a seven?

File: public/emojis.js [hunk 2]
```diff
@@ -84,57 +97,73 @@ const emojis = {
...
   CLOVER: "🍀",
   CLOWN: "🤡",
   CLUB: "♣️",
   COAT: "🧥",
+  COCO: "🥥",
+  COCOA: "🍫",
+  CODER: "👩‍💻",
   COFFIN: "⚰️",
   COIN: "🪙",
+  COLOR: "",
```
Comment (canadianveggie) [inline public/emojis.js:124]: ???

File: public/emojis.js [hunk 3]
```diff
@@ -214,8 +259,11 @@ const emojis = {
   HOTDOG: "🌭",
   HOTEL: "🏨",
   HOUSE: "🏠",
+  HUCK: "🥏",
```
Comment (canadianveggie) [inline public/emojis.js:262]: or DISC?

File: public/emojis.js (hunk 1 above)
Comment (canadianveggie) [reply public/emojis.js:548]: It's very elusive.

If you can find a 4-6 letter alternative, I'm all for it.

File: public/emojis.js [hunk 4]
```diff
@@ -434,43 +533,56 @@ const emojis = {
...
   VOTE: "🗳️",
   WAFFLE: "🧇",
+  WALK: "🚶",
   WARN: "⚠️",
   WATCH: "⌚️",
   WATER: "💧",
   WAVE: "🌊",
   WEBS: "🕸️",
   WHALE: "🐳",
+  WHIP: "🥎",
```
Comment (canadianveggie) [inline public/emojis.js:564]: softball term?

File: public/emojis.js (hunk 1 above)
Comment (canadianveggie) [reply public/emojis.js:548]: https://wordassociations.net/en/words-associated-with/Unicorn
`HORN`
`TUSK`
`RARE`
`MYTH`

or maybe something harder to unlock:
`RARITY`
`HORNED`

File: public/emojis.js (hunk 2 above)
Comment (wachunga) [reply public/emojis.js:124]: thanks for catching this - the build didn't fail 😱 

I'll add a test

File: public/emojis.js (hunk 4 above)
Comment (wachunga) [reply public/emojis.js:564]: ... but maybe that's informal. I guess it's fast-pitch, slow-pitch officially. I'll change it to PITCH

**Review by canadianveggie — APPROVED** [review]

**Pull request merged**